### PR TITLE
Fixes deprecation warnings for Scala 2.13

### PR DESCRIPTION
**Pull request by nvollmar**
References #864

### CODE REVIEW

**Comment by nvollmar** [comment]
@patriknw may I ping you for a review?

**Comment by johanandren** [comment]
OK TO TEST

**Comment by nvollmar** [comment]
Tests were already run, just a review missing 👍

**Comment by johanandren** [comment]
Hehe, yeah, I missed that this isn't a PR in akka/akka when going through my notifications this morning :)

File: core/src/test/scala/akka/persistence/cassandra/CassandraSpec.scala [hunk 1]
```diff
@@ -159,7 +159,7 @@ abstract class CassandraSpec(
         override def write(b: Int): Unit = oldOut.write(b)
       }) {
         override def println(x: Any): Unit =
-          oldOut.println(prefix + String.valueOf(x).replaceAllLiterally("\n", s"\n$prefix"))
+          oldOut.println(prefix + String.valueOf(x).replace("\n", s"\n$prefix"))
```
Comment (johanandren) [inline core/src/test/scala/akka/persistence/cassandra/CassandraSpec.scala:162]: This is not the same, it replaces `\n` with the corresponding position in the second `CharSeq` param, so `\n`. I think it should be `String#replaceAll` rather.

**Review by johanandren — APPROVED** [review]
LGTM with one string replacement remark

File: core/src/test/scala/akka/persistence/cassandra/CassandraSpec.scala (hunk 1 above)
Comment (nvollmar) [reply core/src/test/scala/akka/persistence/cassandra/CassandraSpec.scala:162]: It should be the same as JavaDoc of `replaceAllLiterally` states 
```
Replace all literal occurrences of literal with the literal string replacement. This method is equivalent to #replace.
```

The difference between `replace` and `replaceAll` is that the latter is using a regex.

Comment (johanandren) [reply core/src/test/scala/akka/persistence/cassandra/CassandraSpec.scala:162]: Ah, it is indeed me parsing the JDK javadoc wrong. 👍

**Review by octonato — APPROVED** [review]

**Pull request merged**